keyboards/mitosis/mitosis.h

```diff
@@ -49,19 +49,38 @@
 // This a shortcut to help you visually see your layout.
 // The first section contains all of the arguments
 // The second converts the arguments into a two-dimensional array
+
+// Setting MITOSIS_DATAGROK_BOTTOMSPACE in rules.mk will swap the upper and
+// lower center four thumb-keys. See keymaps/datagrok/rules.mk.
+#ifdef MITOSIS_DATAGROK_BOTTOMSPACE
```

Comment:
This may be something better left to your keymap. 

Eg, you could add this as `LAYOUT_datagrok` in your config.h, and add this to your keymap.c: 

```c
#ifdef MITOSIS_DATAGROK_BOTTOMSPACE
#undef LAYOUT
#define LAYOUT LAYOUT_datagrok
#endf
```

Reply:
Would it be better/worse to leave my `keymap.c` unchanged, and place this in my `config.h`? This would avoid the definition of `LAYOUT_datagrok`.

```c
#ifdef MITOSIS_DATAGROK_BOTTOMSPACE
#undef LAYOUT
#define LAYOUT(\
   ...
) {
   ...
}
#endif
```
This seems simpler to me while still leaving `mitosis.h` unmodified.

Or is perhaps the definition of `LAYOUT_datagrok` useful for QMK configurator?